dev/main.tf

```diff
@@ -1,5 +1,5 @@
 module "dev" {
-  source = "./hcloud-k8s-env"
+  source = "github.com/hetznercloud/terraform-k8s-dev?ref=v0.2.0"
```

Comment:
Should we add renovate here? I think we need a new manager for `.tf` files

Reply:
Nevermind, this should be supported: https://docs.renovatebot.com/modules/manager/terraform/#modules